src/main/java/com/crud/tasks/scheduler/EmailScheduler.java

```diff
@@ -0,0 +1,36 @@
+package com.crud.tasks.scheduler;
+
+import com.crud.tasks.config.AdminConfig;
+import com.crud.tasks.domain.Mail;
+import com.crud.tasks.repository.TaskRepository;
+import com.crud.tasks.service.SimpleEmailService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EmailScheduler {
+    private static final String SUBJECT = "Tasks: Once a day email";
+
+    @Autowired
+    private SimpleEmailService simpleEmailService;
+
+    @Autowired
+    private TaskRepository taskRepository;
+
+    @Autowired
+    private AdminConfig adminConfig;
+
+    @Scheduled(cron = "0 0 10 * * *")
+    //@Scheduled(fixedDelay = 10000)
+    public void sendInformationEmail() {
+        long size = taskRepository.count();
+        String task;
+        if(size == 1) { task = " task";} else { task = " tasks";}
```

Comment:
mozna to zapisac tzw conditional statement 
`String task = size == 1 ? " task" : "tasks"

Reply:
to 1 powinno byc stala. Np private static final SINGLE_TASK